imports/plugins/core/layout/client/templates/layout/wallets/wallet.js

```diff
@@ -0,0 +1,300 @@
+import swal from "sweetalert2";
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+import { Reaction } from "/client/api";
+import { Accounts, Packages, Wallets } from "/lib/collections";
+import "./wallet.css";
+
+let list = [];
+let pageList = [];
+let currentPage = 1;
+const numberPerPage = 10;
+let numberOfPages = 0;
+
+Template.wallet.onCreated(function bodyOnCreated() {
+  this.state = new ReactiveDict();
+  this.state.setDefault({
+    details: { balance: 0, transactions: [] },
+    transactions: [],
+    transactionsList: [],
+    currentPageNum: 1
+  });
+  this.autorun(() => {
+    this.subscribe("transactionDetails", Meteor.userId());
+    const transactionInfo = Wallets.find().fetch();
+    if (transactionInfo.length > 0) {
+      this.state.set("details", transactionInfo[0]);
+      makeList();
+    }
+  });
+});
+
+function makeList() {
+  const transactions = Template.instance().state.get("details").transactions;
+  if (transactions.length > 0) {
+    const newTranscationDetails = transactions.sort(function (a, b) {
+      return new Date(b.date) - new Date(a.date);
+    });
+    Template.instance().state.set("transactionsList", newTranscationDetails);
+    numberOfPages = getNumberOfPages();
+  }
+}
+
+function getNumberOfPages() {
+  list = Template.instance().state.get("transactionsList");
+  if (list.length > 0) {
+    return Math.ceil(list.length / numberPerPage);
+  }
+}
+
+function check() {
+  document.getElementById("next").disabled = currentPage === numberOfPages
+    ? true
+    : false;
+  document.getElementById("previous").disabled = currentPage === 1
+    ? true
+    : false;
+  document.getElementById("first").disabled = currentPage === 1 ? true : false;
```

Comment:
You dont need to write this like this. `currentPage === 1` already returns true or false, so, just do ` document.getElementById("first").disabled = currentPage === 1` and you will be good. Apply this to all other places.